ledger-core/server/internal/headless/components_test.go

```diff
@@ -20,7 +21,12 @@ import (
 )
 
 func TestComponents(t *testing.T) {
-	instestlogger.SetTestOutputWithIgnoreAllErrors(t)
+	instestlogger.SetTestOutputWithErrorFilter(t, func(s string) bool {
+		if strings.Contains(s, "Failed to export to Prometheus: cannot register the collector: duplicate metrics collector registration attempted") {
```

Comment:
also happens
"Failed to export to Prometheus: cannot register the collector: descriptor Desc ..."
so I suggest to use 
"Failed to export to Prometheus: cannot register the collector"